--- a/mft.c
+++ b/mft.c
@@ -620,11 +620,11 @@
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 6, 0)
 	bio->bi_iter.bi_sector =
-		ntfs_bytes_to_bio_sector(vol, NTFS_CLU_TO_B(vol, vol->mftmirr_lcn) +
+		ntfs_bytes_to_bio_sector(NTFS_CLU_TO_B(vol, vol->mftmirr_lcn) +
 				 lcn_folio_off + folio_ofs);
 #else
 	bio->bi_iter.bi_sector =
-		ntfs_bytes_to_bio_sector(vol, NTFS_CLU_TO_B(vol, vol->mftmirr_lcn) +
+		ntfs_bytes_to_bio_sector(NTFS_CLU_TO_B(vol, vol->mftmirr_lcn) +
 				 lcn_page_off + page_ofs);
 #endif
 
@@ -758,7 +758,7 @@
 #else
 		bio = bio_alloc(vol->sb->s_bdev, 1, REQ_OP_WRITE, GFP_NOIO);
 		bio->bi_iter.bi_sector =
-			ntfs_bytes_to_bio_sector(vol, NTFS_CLU_TO_B(vol, ni->mft_lcn[i]) +
+			ntfs_bytes_to_bio_sector(NTFS_CLU_TO_B(vol, ni->mft_lcn[i]) +
 					 clu_off);
 		if (!bio) {
 			err = -ENOMEM;
